packages/wc-compass/src/layouts/about/_about.scss

```diff
@@ -0,0 +1,81 @@
+div.cdg-about-section {
```

Comment:
Don't use div. selector like this, the name should be .cdg-section-about, in the future there will be .cdg-section... other.

Reply:
resolved